src/mon/NVMeofGwMap.cc

```diff
@@ -430,6 +430,62 @@ void  NVMeofGwMap::find_failover_candidate(
   }
 }
 
+void  NVMeofGwMap::handle_gw_performing_fast_reboot(const NvmeGwId &gw_id,
+     const NvmeGroupKey& group_key, bool &map_modified)
+{
+  bool modified = false;
+  for (auto& state_itr: created_gws[group_key][gw_id].sm_state ) {
+    fsm_handle_gw_fast_reboot(gw_id,group_key, state_itr.first, modified);
+    map_modified |= modified;
+  }
+}
+
+void NVMeofGwMap::fsm_handle_gw_fast_reboot(const NvmeGwId &gw_id,
+    const NvmeGroupKey& group_key, NvmeAnaGrpId grpid, bool &map_modified) {
```

Comment:
Brace on new line for definitions in .cc files as you did in handle_gw_performing_fast_reboot above.